webpack.config.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
   plugins: [
     new BundleTracker({ filename: './webpack-stats.json' }),
     new ExtractTextPlugin('[name]-[hash].css'),
+    new webpack.ProvidePlugin({
+      'Promise': 'exports?global.Promise!es6-promise',
```

Comment:
Do we actually need this? Self-repair isn't meant to work outside of Firefox, and it was introduced in, like, Firefox 30-something (@gregglind knows for sure) where native Promises were already a thing . The preview will also eventually require the system add-on to be installed, so trying to make it compatible with other browsers will eventually fail.